### ISSUE TITLE
Mythbox on Windows cannot find Mythtv recordings directory

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#38]
```
What steps will reproduce the problem?
1. Install mythbox script in Windows XP SP3 according to the readme
instructions.
2. Insert mythtv backend ip address and mythconverg password.
3. Select 'test settings' in mythbox setup.

What is the expected output? What do you see instead?
Expected output is that test was successful and mythbox starts to work.
Actual output is 'Recording directory /var/lib/mythtv/recordings does not
exist'

What version of the product are you using? On what operating system?
Alpha release of mythbox(SVN 682). Windows XP SP3. XBMC for windows 9.04.1.

Please provide any additional information below.
Mythbox working fine on my combined backend/ frontend linux box (where the
recordings folder resides) using the same settings as input onto the
windows version of the script.

I have a symlink from /var/lib/mythtv/recordings to another location (on
another partition of the same linux box) where the actual recordings are
stored. Inputting this location as the recordings path returns the same
error message.

```

Original issue reported on code.google.com by `bklein1` on 3 Jun 2009 at 11:16

**Comment by GoogleCodeExporter** [comment]
```
I forgot to mention that the built-in XBMC myth support (the myth:// protocol) 
works
perfectly well on this XP machine and has no trouble accessing the recordings.
```

Original comment by `bklein1` on 4 Jun 2009 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
On your windows box, you have to mount the remote directory that recordings are
stored in. This would most likely map to a drive letter and this is what you 
would
enter for the 'Recordings Directory' settings in Mythbox.
```

Original comment by `semir.pa...@gmail.com` on 9 Jun 2009 at 10:35
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
I ran into this also. I mapped "R:" to \\mythtv\recordings and I can browse that
fine, however if I try to stick "R:\" in as the recordings folder setting, 
Mythbox
says it doesn't exist.

Oddly though when I browse recordings in Mythbox it's pulling up the PNG from 
that
folder but not the video 

*confused*
```

Original comment by `ccare...@gmail.com` on 25 Sep 2009 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Can you try "R:" instead?
```

Original comment by `semir.pa...@gmail.com` on 25 Sep 2009 at 8:52

**Comment by GoogleCodeExporter** [comment]
```
I have mapped  \\mythtv\recordings videos and livetv to r: v: and l:.  I tried 
r:\;l:\;v:\; and r:;l;v;.  Both give sound but no video in both recorded or 
live tv.  Also tried capital letters as well.  I have windows 7, 64 bit and 
mythbuntu release-0-23-fixes [24158] on mythbuntu 10.04 lucid.

The following is an excerpt from backend log while starting up mythbox script 
and then trying to watch recording.


2010-09-03 22:31:19.220 MainServer::HandleVersion - Client speaks protocol 
version 8 but we speak 56!
2010-09-03 22:31:19.238 MainServer, Warning: Unknown socket closing 
MythSocket(0xffffffffabed4870)
2010-09-03 22:31:19.243 MainServer::ANN Playback
2010-09-03 22:31:19.255 adding: Lenovo as a client (events: 0)
2010-09-03 22:31:20.597 MainServer::ANN Playback
2010-09-03 22:31:20.600 adding: Lenovo as a client (events: 0)
2010-09-03 22:31:20.622 MainServer::ANN Playback
2010-09-03 22:31:20.625 adding: Lenovo as a client (events: 0)
2010-09-03 22:31:20.668 MainServer::ANN Playback
2010-09-03 22:31:20.709 adding: Lenovo as a client (events: 0)
2010-09-03 22:31:35.061 ProgramInfo(): Updated pathname '':'' -> 
'2048_20100903190000.mpg'
2010-09-03 22:31:40.237 ProgramInfo(): Updated pathname '':'' -> 
'2048_20100903190000.mpg'


Excerpt from C:\Users\username\AppData\Roaming\XBMC

....
2:33:45 T:1472 M:3872129024   ERROR: D3DXCreateEffectFromFile texture 
g_YTexture;
                                            texture g_UTexture;
                                            texture g_VTexture;
                                            float4x4 g_ColorMatrix;

                                            sampler YSampler =
                                              sampler_state {
                                                Texture = <g_YTexture>;
                                                AddressU = CLAMP;
                                                AddressV = CLAMP;
                                                MipFilter = LINEAR;

So I decided to just add r:\ to the videos and watch that without using 
mythbox.... guess what .... no video....  Thus I think that the problem is 
related to the original error that I found of "Error d3dx9_42.dll not found", 
which I thought I fixed by adding the dll into the xmbc folder.  
(http://forum.xbmc.org/showthread.php?t=67711)  Duah, not the right thing to do.

So obviously this is not the right place to post this... So why.... To help 
others... If anyone knows how to properly install the missing directx dll in 
windows 7, please let me and others know.  Thanks.

```

Original comment by `mysmar...@gmail.com` on 4 Sep 2010 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
got it... everyone on the link that I posted was linking to the directx 
redistributable, not the installer.  
http://www.microsoft.com/downloads/en/confirmation.aspx?familyId=2da43d38-db71-4
c1b-bc6a-9b6652cd92a3&displayLang=en
```

Original comment by `mysmar...@gmail.com` on 4 Sep 2010 at 3:55
